webapp/login_test.go

```diff
@@ -0,0 +1,104 @@
+// +build medium
+// Copyright 2019 The WPT Dashboard Project. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package webapp
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/google/go-github/v28/github"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/golang/mock/gomock"
+	"github.com/web-platform-tests/wpt.fyi/shared"
+	"github.com/web-platform-tests/wpt.fyi/shared/sharedtest"
+)
+
+func TestHandleLogin(t *testing.T) {
+	mockCtrl := gomock.NewController(t)
+	defer mockCtrl.Finish()
+	ctx := sharedtest.NewTestContext()
+	w := httptest.NewRecorder()
+	req := httptest.NewRequest("GET", "https://foo/login", nil)
+
+	mockStore := sharedtest.NewMockDatastore(mockCtrl)
+	mockStore.EXPECT().NewNameKey("Token", gomock.Any()).AnyTimes().Return(nil)
+	mockStore.EXPECT().Get(gomock.Any(), gomock.Any()).AnyTimes().Return(nil).Do(func(key shared.Key, dst interface{}) {
+		token, _ := dst.(*shared.Token)
+		// Need to create a 16 bytes fake secrete for securecookie.
```

Comment:
Clarify that the length is the important bit, and the actual value doesn't matter.